src/selectedtests/task_mappings/mappings.py

```diff
@@ -225,17 +227,19 @@ def _map_tasks_to_files(changed_files: List[str], flipped_tasks: Dict, task_mapp
                 builds_to_task_mappings[cur_task] = cur_flips_for_task + 1
 
 
-def _filter_non_required_distros(builds: List[Build]) -> List[Build]:
+def _filter_non_required_distros(builds: List[Build], build_regex: Pattern) -> List[Build]:
     """
     Filter the distros that aren't required in evergreen.
```

Comment:
I think this method name would be clearer if we replaced "required" with "matching"

Reply:
Sure